src/compiler/diagnosticMessages.json

```diff
@@ -3305,6 +3305,10 @@
         "category": "Error",
         "code": 5077
     },
+    "'{0}' could be instantiated with an arbitrary type which could be unrelated to '{1}'.": {

```

Comment:
Does `could be instantiated with an arbitrary type which **is** be unrelated to` read better?

Reply:
I think it's fine as it is